.github/workflows/isotovideo.yml

```diff
@@ -1,11 +1,49 @@
 ---
 name: isotovideo
 # yamllint disable-line rule:truthy
-on: [push, pull_request]
+on:
+  - push
+  - pull_request
 jobs:
-  test:
-    runs-on: ubuntu-latest
+  isotovideo:
+    strategy:
+      matrix:
+        # An example matrix, it'll create a derivate job for each
+        # item specified here. For simplicity sake the example below
+        # has the full var line since it's easy enough.
+        #
+        # For more complex setups it'll be worth exploring advance matrix setups
+        # see: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-expanding-configurations
+        isotovideo-args:
+          - "qemu_no_kvm=1 casedir=."
+    runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
-      - name: Run isotovideo against test code in happy-path scenario
-        run: podman run --rm -it -v .:/tests:Z registry.opensuse.org/devel/openqa/containers/isotovideo:qemu-x86 qemu_no_kvm=1 casedir=/tests
```

Comment:
As explained in https://github.com/os-autoinst/os-autoinst-distri-example/pull/28 on multiple places, in particular https://github.com/os-autoinst/os-autoinst-distri-example/pull/28#issuecomment-1864431251 , I am not convinced we should do that change from a single podman call to a github workflow specific syntax. There are of course some advantages but as disadvantages I see that people can not just copy-paste that line to reproduce. I would go even further and extract that line into a Makefile so that github CI same as users can just call `make test-isotovideo-happy-path` without duplication.